src/sdk/model/cloud-tenant.ts

```diff
@@ -0,0 +1,166 @@
+import { Entity } from './entity';
+import { CloudTenantJson, VeeamResource } from './json/cloud-tenant';
+import { Iland } from '../iland';
+import { EntityType } from './json/entity-type';
+
+/**
+ * Cloud Tenant.
+ */
+export class CloudTenant extends Entity {
+
+  constructor(private _json: CloudTenantJson) {
+    super(_json);
+  }
+
+  /**
+   * Gets a Cloud Tenant by UUID.
+   * @param {string} uuid
+   * @returns {Promise<CloudTenant>} promise that resolves with the Cloud Tenant
+   */
+  static async getCloudTenant(uuid: string): Promise<CloudTenant> {
+    return Iland.getHttp().get(`/cloud-tenants/${uuid}`).then((response) => {
+      const json = response.data as CloudTenantJson;
+      return new CloudTenant(json);
+    });
+  }
+
+  get entityType(): EntityType {
+    return 'VCC_TENANT';
+  }
+
+  /**
+   * Gets the uid of the Cloud Tenant.
+   * @returns {string} id value
+   */
+  get uid(): string {
+    return this._json.uid;
+  }
+
+  /**
+   * Indicates whether the Cloud Tenant is enabled.
+   * @returns {boolean} value
+   */
+  get enabled(): boolean {
+    return this._json.enabled;
+  }
+
+  /**
+   * Gets the VeeamResource of the Cloud Tenant.
+   * @returns {VeeamResource} value
+   */
+  get resources(): VeeamResource {
```

Comment:
There is no reason to make the javascript/typescript representation match the representation in the JSON. In fact, one of the main reasons for writing these wrapper classes is to make the Javascript/Typescript client interface better than the interface provided by the API. This is a perfect case actually.

Reply:
@cfsnyder pr updated